### ISSUE TITLE
Accessibility: Tab to items

### ISSUE THREAD

**Issue by niemyjski** [huntabyte/cmdk-sv#72]
Shouldn't command items be tabbed by default for accessibility? You can only use keyboard shortcuts when on them, but how would someone know that?

**Comment by huntabyte** [comment]
To answer your points:

1. Command items should not be tabbed, you navigate Combobox menus with arrow keys.
![CleanShot 2024-04-01 at 10 18 03@2x](https://github.com/huntabyte/cmdk-sv/assets/64506580/24e2cb98-21b9-4b1d-8183-0209687b82f8)
ref: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/
2. The reason you can't navigate anywhere when the combobox is open is that we apply a focus trap that keeps focus within the popover until it is closed either by clicking outside or pressing the escape key.
3. I'm not sure what you're referring to on this one. I can navigate through the list with arrow keys just fine.
4. Pressing `Space` to select items is not acceptable behavior for a Combobox. Space is a valid character that could be typed to further filter. Only `Enter` selects the items.

**Comment by niemyjski** [comment]
3. I'm doing what you were doing in your status tasks facet dropdown for checkboxes but nothing is selectable. I'll try to see if updating it to be closer to your updated sample fixes it.

```

                            <Command.Item id={option.value} value={option.value} onSelect={() => onValueSelected(option.value)}>
                                <div
                                    class={cn(
                                        'mr-2 flex h-4 w-4 items-center justify-center rounded-sm border border-primary',
                                        $updatedValues.includes(option.value) ? 'bg-primary text-primary-foreground' : 'opacity-50 [&_svg]:invisible'
                                    )}
                                >
                                    <IconCheck className={cn('h-4 w-4')} />
                                </div>
                                <span>
                                    {option.label}
                                </span>
                            </Command.Item>
```

**Comment by huntabyte** [comment]
If you can link a minimal repro I'm happy to take a look as well to see what's causing the issue. It may be an ordering issue of the components or something!

**Comment by niemyjski** [comment]
Yes, I'll work on a sample. We've just had massive power outages the past 2 days.

**Comment by niemyjski** [comment]
@huntabyte thanks for your patience, I've created a sample here. Just run `npm install` and `npm run dev` 

https://github.com/exceptionless/exceptionless-svelte-shadcn-cmdk-sv-repo

On a side note what is the best way to be more productive for you to reproduce issues. Should we fork the project and create a pr with maybe story book or tests or would a template repo where everything is configured and we just fork be easier for you in the future?

**Comment by pczern** [comment]
3. also doesn't work for me, sometimes it navigates 1 item with up/down keys and then it stops, it also doesn't hover anymore then.
I am using Svelte 5 Preview.
